### ISSUE TITLE
news - image and text with redundant links

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#101]
The article author photo and the name links are coming up as redundant links.

**Comment by kimisgold** [comment]
This should be ready for another look.

**Comment by IdaAkiwumi** [comment]
WAVE is alerting to images having nearby images with the same alt text. I'm not sure what happened, but I'm noticing a lot more of these alerts. 

![Screen Shot 2020-10-19 at 12 06 31 PM](https://user-images.githubusercontent.com/618307/96478091-4368a500-1205-11eb-8d87-0b913ff9c18c.png)
